sql/driver/rows.go

```diff
@@ -22,12 +22,24 @@ import (
 	"io"
 )
 
+type row []driver.Value
+
 type rows struct {
 	columns []string
-	// TODO(pmattis): This is unfinished. Each row needs to have len(columns)
-	// entries.
-	rows []string
-	pos  int // Current iteration index into rows.
+	rows    []row
+	pos     int // Current iteration index into rows.
+}
+
+func newSingleColumnRows(column string, vals []string) *rows {
```

Comment:
Honestly, I had to read the source to figure out what this is doing. The name is pretty inscrutable. Either come up with a better name or add a comment.